lib/ts/recipe/passwordless/recipe.tsx

```diff
@@ -75,14 +75,14 @@ export default class Passwordless extends AuthRecipe<
             const normalisedFullPath = this.config.appInfo.websiteBasePath.appendPath(new NormalisedURLPath("/"));
             features[normalisedFullPath.getAsStringDangerous()] = {
                 matches: matchRecipeIdUsingQueryParams(this.config.recipeId),
-                component: (props) => this.getFeatureComponent("signInUp", props),
+                component: (props) => this.getFeatureComponent("signInUp", props as any),
```

Comment:
Why are these changes required?

Reply:
just to make typescript stop yelling, file is always marked as "with errors" without. these changes